### PR TITLE
[Magic Transit] Added section for prefix auto advertisement

### PR DESCRIPTION
**Pull request by dcpena**
Added documentation for prefix auto advertisement to address [PCX-3530](https://jira.cfops.it/browse/PCX-3530).

### CODE REVIEW

File: content/magic-transit/how-to/auto-advertise-prefixes.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+         "automatic_advertisement": true,
+         "duration": "1m0s",
+        ]
+```
+
+For more granular thresholds, create a more focused rule as shown below.
+
+```txt
+"rules":[
+         "name": "Too many packets",
```
Comment (kodster28) [inline content/magic-transit/how-to/auto-advertise-prefixes.md:30]: Should this be the same name as the one above? I thought this had less packets

File: content/magic-transit/how-to/auto-advertise-prefixes.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Notifications
+
+Webhook, Pagerduty, and Email notifications are sent following an auto-advertisement attempt for all prefixes inside the flagged rule.
+
+You will receive the status of the advertisement for each prefix with the following available statuses:
+
+- **Advertised** - The prefix was successfully advertised.
+- **Already Advertised** - The prefix was advertised prior to the auto advertisement attempt.
+- **Delayed** - The prefix cannot currently be advertised but will attempt advertisement. After the prefix can be advertised, a new notification is sent with the updated status.
+- **Locked** - The prefix is locked and cannot be advertised
```
Comment (kodster28) [inline content/magic-transit/how-to/auto-advertise-prefixes.md]:
```suggestion
- **Locked** - The prefix is locked and cannot be advertised.
```

File: content/magic-transit/how-to/enable-flow-based-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+pcx-content-type: how-to
+title: Enable Flow-based monitoring alerts
+---
+
+## Enable Flow-based monitoring alerts
+
+1.  Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Notifications**.
+2.  From **Notifications**, click **Add**.
+3.  Locate **Magic Transit** in the list and click **Select** to add a Flow-based Monitoring: Volumetric Attack notification.
```
Comment (kodster28) [inline content/magic-transit/how-to/enable-flow-based-monitoring.md]: some formatting missing for `flow-based monitoring: volumetric attack notification`?

File: content/magic-transit/how-to/auto-advertise-prefixes.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        ]
+```
+
+## Auto advertisement for prefixes inside a flagged rule
+
+To enable an auto advertisement flag for each created rule, contact your account team. Auto advertisement allows Magic Transit customers to automatically advertise all prefixes inside a rule whenever it is flagged by an incoming attack.
+
+## Notifications
+
+Webhook, Pagerduty, and Email notifications are sent following an auto-advertisement attempt for all prefixes inside the flagged rule.
```
Comment (kodster28) [inline content/magic-transit/how-to/auto-advertise-prefixes.md:44]: Do you have to do anything special to enable these? Thinking maybe it makes sense to link to the enable flow-based monotirong?

**Review by kodster28 — APPROVED** [review]

File: content/magic-transit/how-to/auto-advertise-prefixes.md (hunk 1 above)
Comment (dcpena) [reply content/magic-transit/how-to/auto-advertise-prefixes.md:30]: This was the example that was provided and the name is the same, so I kept it as is. Leaving it for now but will double-check

File: content/magic-transit/how-to/auto-advertise-prefixes.md (hunk 4 above)
Comment (dcpena) [reply content/magic-transit/how-to/auto-advertise-prefixes.md:44]: Nope, if my understanding is correct, they just happen automatically.

**Pull request merged**